api/converter/converter_test.go

```diff
@@ -26,6 +26,7 @@ import (
 	"github.com/yorkie-team/yorkie/api/converter"
 	"github.com/yorkie-team/yorkie/pkg/document"
 	"github.com/yorkie-team/yorkie/pkg/document/proxy"
+	time2 "github.com/yorkie-team/yorkie/pkg/document/time"
```

Comment:
How about using imports as `time` and `gotime`?

```suggestion
	gotime "time"
	...
	"github.com/yorkie-team/yorkie/pkg/document/time"
```

Reply:
Thanks for the feedback!
This proposal has been modified from b52d78d. Please check.